$OpenBSD: patch-aclocal_m4,v 1.3 2002/11/28 20:35:43 margarida Exp $

XXX: A dirty hack of AUTOCONF_NEW, should not be needed when espie@
     upgrading autoconf-new to 2.54 in the ports tree. -- jim
     
--- aclocal.m4.orig	Sun Nov 24 21:36:45 2002
+++ aclocal.m4	Sun Nov 24 21:36:53 2002
@@ -70,7 +70,7 @@ AU_DEFUN([AM_CONFIG_HEADER], [AC_CONFIG_
 # CC etc. in the Makefile, will ask for an AC_PROG_CC use...
 
 
-AC_PREREQ([2.54])
+AC_PREREQ([2.52])
 
 # Autoconf 2.50 wants to disallow AM_ names.  We explicitly allow
 # the ones we care about.
